templates/visuals/default/src/visual.ts

```diff
@@ -35,12 +35,23 @@ module powerbi.extensibility.visual {
             console.log('Visual constructor', options);
             this.target = options.element;
             this.updateCount = 0;
+
         }
 
         public update(options: VisualUpdateOptions) {
             this.settings = Visual.parseSettings(options && options.dataViews && options.dataViews[0]);
             console.log('Visual update', options);
-            this.target.innerHTML = `<p>Update count: <em>${(this.updateCount++)}</em></p>`;
+            while (this.target.firstChild) {
```

Comment:
I'm not sure that we need all these code. We might construct DOM once and just update the text node in the update method.